integration/account/cmd_account_test.go

```diff
@@ -0,0 +1,94 @@
+package account_test
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/ignite-hq/cli/ignite/pkg/cmdrunner/step"
+	"github.com/ignite-hq/cli/ignite/pkg/randstr"
+	envtest "github.com/ignite-hq/cli/integration"
+	"github.com/stretchr/testify/require"
+)
+
+const testAccountMnemonic = "develop mansion drum glow husband trophy labor jelly fault run pause inside jazz foil page injury foam oppose fruit chunk segment morning series nation"
+
+func TestAccount(t *testing.T) {
```

Comment:
I just realized there were no integration tests for account, so I added some (they are really fast)